docker-in-action.md

```diff
@@ -45,17 +45,17 @@ this uses two flags, `--interactive` (or `-i`) and `--tty` (or `-t`).
 
 The `--interactive` flag tells Docker to keep the the *standard Input stream* open. 
 
-The `-t` flag tells docker to *allocate a virtual terminal for the container*, Which will  allow you ot pass the signals to the container.
+The `-t` flag tells docker to *allocate a virtual terminal for the container*, Which will  allow you to pass the signals to the container.
 
-The command in teh interactive container exampel creates a container, starts a UNIX shell, and is linked to the container that's running NGINX. From this shell we can verify that the web server is running correctly by 
+The command in the interactive container example creates a container, starts a UNIX shell, and is linked to the container that's running NGINX. From this shell we can verify that the web server is running correctly by 
 
 `
 wget -O - http://web:80/
 `
 
 This terminal can be shut down by typing  `exit`. **This will terminate the shell program and stop the container.**
 
-It's possible to create an interactive container, manyally start a process insisde that container and then detach you terminal. You can do so by holding Ctrl (or Control) key and pressing P and then Q. This will work only when you've used the --tty option.
+It's possible to create an interactive container, manually start a process inside that container and then detach you terminal. You can do so by holding Ctrl (or Control) key and pressing P and then Q. This will work only when you've used the --tty option.
 
 This is a monitoring agent that will test the web server as we did previously and send a message with the mailer if the web server stops. this command will start the agent in a interactive container using the short-form flags:
```

Comment:
does this mean we can use it as an alerting mechanism? will it send a unix mail or web mail?